samples/plugins/mysql_odbc/connectionResolver.tdr

```diff
@@ -6,6 +6,7 @@
     </connection-builder>
     <connection-normalizer>
       <required-attributes>
+        <setImpersonateAttributes/>
```

Comment:
Did we have any documentation about what this is or why someone would want to set it?

Reply:
I have some doc changes in the pipe as a side thing for hackathon that include this scenario.